src/index.ts

```diff
@@ -117,6 +117,7 @@ export interface ReleasePRConstructorOptions
   extends ReleasePROptions,
     ReleaserConstructorOptions {
   labels?: string[];
+  forManifestReleaser?: boolean;
```

Comment:
Can we rename this to what the option is actually trying to do (like `skipDependencyUpdates` or something like that)?

In the future, we (I) would like to make all releasers manifest releasers with a single releaser as an expected use-case. It would also make it a bit clearer what the option is trying to do.